src/routes/opening/index.jsx

```diff
@@ -88,6 +88,12 @@ const BodyFooter = styled.div`
   align-items: flex-end;
 `
 
+const BackButton = styled(Button)`
```

Comment:
We should avoid to create custom componentes on the fly and try to use the componentes we already have.

If this is a super particular component, we should talk with the designers team and analyze if it's worth to add the new component or it's better to re-use an existing one.

Reply:
What do you mean by custom components? It's a wrapper on button component with positioning styles